crates/bevy_audio/src/audio_output.rs

```diff
@@ -43,7 +43,8 @@ where
 {
     fn play_source(&self, audio_source: &Source) {
         if let Some(stream_handle) = &self.stream_handle {
-            let sink = Sink::try_new(stream_handle).unwrap();
+            let sink =
+                Sink::try_new(stream_handle).expect("Could not create Sink from stream_handle.");
```

Comment:
```suggestion
                Sink::try_new(stream_handle).expect("Could not play audio");
```

`Sink::try_new` will return a https://docs.rs/rodio/0.15.0/rodio/enum.PlayError.html, Rust will have the message in expect followed by `: PlayError::debug()`, so the message shouldn't end with a `.`.

It's also possible to get this error without having touched at a sink or knowing what it is